### ISSUE TITLE
Allowed hours seems ignored in 2.2.3

### ISSUE THREAD

**Issue by serl** [lavinir/hassio-onedrive-backup#145]
Hello,
I set allowed hours to `4-6`, and I got a backup today at 13:03.
This happens since I updated from 2.1.2 to 2.2.3 (around 10 days ago, but I didn't notice before).

(I had to migrate my settings manually, as the automatic migration did not work, not sure this could be a hint or not)

Thanks!

**Comment by lavinir** [comment]
Could you please restart the addon, it should output to the logs the allowed hours it recognizes and share the logs?

**Comment by serl** [comment]
Here's the log. Looks good!
I guess I'll reopen in 3 days if it doesn't work :)

```
11/08/2023 17:43:01 [1] Info: Anonymous Telemetry Disabled
11/08/2023 17:43:01 [1] Info: Backup interval configured to every 72 hours
11/08/2023 17:43:01 [1] Info: Generational backups enabled
11/08/2023 17:43:01 [1] Info: Backups / Syncs will only run during these hours: 4,5,6
```

**Comment by lavinir** [comment]
Let's do it another way. I'll keep it open you you can close if this is resolved.

**Comment by lavinir** [comment]
Ok I took a better look at this. Looks like there's a bug if you update the allowed hours without restarting the addon it doesn't take effect. Will be fixed in the next update.

**Comment by lavinir** [comment]
Fixed in 2.2.4
